src/BalconyBotanica/Adapters/Database/PlantRepository.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BalconyBotanica.Adapters.Database.DummyDatabase;
+using BalconyBotanica.Core.DomainObjects;
+
+namespace BalconyBotanica.Adapters.Database
+{
+    //  data access concerns only.
+    public class PlantRepository : IPlantRepository
+    {
+        DummyDatabase.DummyDatabase dummyPlantData = new();
+
+        public PlantDataDbo GetPlantById(int plantId)
+        {
+            return dummyPlantData.ReturnDummyPlant();
```

Comment:
This always returns `plantData1` but I guess you just haven't finished the implementation?

Reply:
Correct, this was just so it returned something :) I'll add "TODO: " in my new code when something isn't finished.